<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>491</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qawno.qrc">
    <normaloff>:/icons/pawn.ico</normaloff>:/icons/pawn.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="EditorWidget" name="editor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>3</verstretch>
        </sizepolicy>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="OutputWidget" name="output">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>694</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionFindNext"/>
    <addaction name="separator"/>
    <addaction name="actionGoToLine"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>&amp;Build</string>
    </property>
    <addaction name="actionCompile"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <widget class="QMenu" name="menuFont">
     <property name="title">
      <string>&amp;Font</string>
     </property>
     <addaction name="actionEditorFont"/>
     <addaction name="actionOutputFont"/>
    </widget>
    <widget class="QMenu" name="menuIndentation">
     <property name="title">
      <string>&amp;Indentation</string>
     </property>
     <addaction name="actionUseTabs"/>
     <addaction name="actionUseSpaces"/>
     <addaction name="separator"/>
     <addaction name="actionTabWidth2"/>
     <addaction name="actionTabWidth4"/>
     <addaction name="actionTabWidth8"/>
     <addaction name="separator"/>
     <addaction name="actionIndentWidth2"/>
     <addaction name="actionIndentWidth4"/>
     <addaction name="actionIndentWidth8"/>
    </widget>
    <addaction name="menuFont"/>
    <addaction name="menuIndentation"/>
    <addaction name="separator"/>
    <addaction name="actionCompiler"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuBuild"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>Create a new file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Open and existing file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save changes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="toolTip">
    <string>Save changes to a different file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Close the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>&amp;Find...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionFindNext">
   <property name="text">
    <string>Find &amp;Next</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionCompile">
   <property name="text">
    <string>&amp;Compile</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionCompiler">
   <property name="text">
    <string>&amp;Compiler</string>
   </property>
   <property name="toolTip">
    <string>Compiler settings</string>
   </property>
  </action>
  <action name="actionEditorFont">
   <property name="text">
    <string>&amp;Editor</string>
   </property>
   <property name="toolTip">
    <string>Change editor font</string>
   </property>
  </action>
  <action name="actionOutputFont">
   <property name="text">
    <string>&amp;Output</string>
   </property>
   <property name="toolTip">
    <string>Change font of the output window</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="toolTip">
    <string>Finish editing the current file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionGoToLine">
   <property name="text">
    <string>Go To Line...</string>
   </property>
   <property name="toolTip">
    <string>Jump to the specified line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionUseTabs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Use &amp;Tabs</string>
   </property>
  </action>
  <action name="actionUseSpaces">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use &amp;Spaces</string>
   </property>
  </action>
  <action name="actionIndentWidth2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Indent Width: 2</string>
   </property>
  </action>
  <action name="actionIndentWidth4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Indent Width: 4</string>
   </property>
  </action>
  <action name="actionIndentWidth8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Indent Width: 8</string>
   </property>
   <property name="toolTip">
    <string>Indent Width: 8</string>
   </property>
  </action>
  <action name="actionTabWidth2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tab Width: 2</string>
   </property>
  </action>
  <action name="actionTabWidth4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tab Width: 4</string>
   </property>
  </action>
  <action name="actionTabWidth8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tab Width: 8</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EditorWidget</class>
   <extends>QPlainTextEdit</extends>
   <header>src/EditorWidget.h</header>
  </customwidget>
  <customwidget>
   <class>OutputWidget</class>
   <extends>QPlainTextEdit</extends>
   <header>src/OutputWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qawno.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionUndo</sender>
   <signal>triggered()</signal>
   <receiver>editor</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRedo</sender>
   <signal>triggered()</signal>
   <receiver>editor</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCut</sender>
   <signal>triggered()</signal>
   <receiver>editor</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCopy</sender>
   <signal>triggered()</signal>
   <receiver>editor</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPaste</sender>
   <signal>triggered()</signal>
   <receiver>editor</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
